The Debian packaging of farbfeld is maintained in git, using the merging
workflow described in dgit-maint-merge(7).  There isn't a patch
queue that can be represented as a quilt series.

A detailed breakdown of the changes is available from their
canonical representation - git commits in the packaging repository.

    % git clone https://git.dgit.debian.org/farbfeld
    % cd foo
    % git log --oneline 2.20170109-1..debian/2.20170109 -- . ':!debian'

(If you have dgit, use `dgit clone foo', rather than plain `git
clone'.)

A single combined diff, containing all the changes, follows.
--- farbfeld-2.20170109.orig/2ff
+++ farbfeld-2.20170109/2ff
@@ -24,6 +24,11 @@ image/jpeg)
 	jpg2ff < "$TMP"
 	;;
 *)
+	if [ ! -x /usr/bin/convert ] ; then
+		printf "%s: cant convert from %s -- missing \`convert' program" "$0" "$FORMAT" >&2
+		printf "Run \`apt-get install imagemagick' to install it" >&2
+		exit 1
+	fi
 	convert "$TMP" png:- 2>/dev/null | png2ff 2>/dev/null
 	;;
 esac
--- farbfeld-2.20170109.orig/config.mk
+++ farbfeld-2.20170109/config.mk
@@ -4,8 +4,8 @@ VERSION = 2
 # Customize below to fit your system
 
 # paths
-PREFIX = /usr/local
-MANPREFIX = ${PREFIX}/man
+PREFIX = /usr
+MANPREFIX = ${PREFIX}/share/man
 
 PNGLIB = /usr/local/lib
 PNGINC = /usr/local/include
@@ -17,9 +17,9 @@ INCS =
 LIBS =
 
 # flags
-CPPFLAGS = -D_DEFAULT_SOURCE
-CFLAGS   = -std=c99 -pedantic -Wall -Os ${INCS}
-LDFLAGS  = -s ${LIBS}
+CPPFLAGS += -D_DEFAULT_SOURCE
+CFLAGS   += -std=c99 -pedantic -Wall -Os ${INCS}
+LDFLAGS  += -s ${LIBS}
 
 # compiler and linker
 CC = cc
